### ISSUE TITLE
Codegen cli crash when Fetching latest plugins

### ISSUE THREAD

**Issue by ludovicjamet** [dotansimha/graphql-code-generator#7997]
### Describe the bug

Crash when launching the init script

```
Fetching latest versions of selected plugins...
require() of ES Module /placeholder/node_modules/latest-version/index.js from /placeholder/node_modules/@graphql-codegen/cli/bin.js not supported.
```

### Your Example Website or App

Not possible it's about the cli command

### Steps to Reproduce the Bug or Issue

On a fresh Next JS project bootstrapped with `npx create-next-app@latest --typescript` after the installation of the following yarn packages : 

```
 "@apollo/client": "^3.6.9",
 "@graphql-codegen/cli": "^2.6.3",
 "@graphql-codegen/typescript": "^2.5.1",
 "@graphql-codegen/typescript-operations": "^2.4.2",
 "@graphql-codegen/typescript-react-apollo": "^3.2.16",
 "graphql": "^16.5.0",
 "next": "12.2.0",
 "react": "18.2.0",
 "react-dom": "18.2.0"

```

Then launch the codegen init command:

`yarn graphql-codegen init
`

### Expected behavior

No crash, the package should be ESM compatible

### Platform

- OS: macOS
- NodeJS: 16.15.0
- `graphql` version: 16.5.0
- `@graphql-codegen/*` version(s):  2.6.3

### Codegen Config File

```
overwrite: true
schema: "http://localhost:4000"
documents: "src/**/*.graphql"
generates:
  src/generated/graphql.tsx:
    plugins:
      - "typescript"
      - "typescript-operations"
      - "typescript-react-apollo"
  ./graphql.schema.json:
    plugins:
      - "introspection"
```

**Comment by ludovicjamet** [comment]
A solution is to downgrade the package to the version 2.6.1

**Comment by charlypoly** [comment]
Hi @ludovicjamet,

we just released a new patch version of the cli that fixes the issue: `@graphql-codegen/cli@2.6.4`
